pkg/apis/ceph.rook.io/v1/object.go

```diff
@@ -76,6 +76,9 @@ func ValidateObjectSpec(gs *CephObjectStore) error {
 	if gs.Namespace == "" {
 		return errors.New("missing namespace")
 	}
+	if len(gs.Name) > 38 {
```

Comment:
Let's make 38 a const variable on the top

Reply:
Added the reason in the comment

Reply:
I'd vote for the const as well, still helpful for code readability.